elide-annotations/pom.xml

```diff
@@ -9,7 +9,7 @@
     <parent>
         <groupId>com.yahoo.elide</groupId>
         <artifactId>elide-parent-pom</artifactId>
-        <version>3.1.3-SNAPSHOT</version>
+        <version>4.0-alpha</version>
```

Comment:
Don't bump the version yet. First, let's rebase or merge master -> elide-4.x. This will save us some merge conflict pain.

Reply:
We'll probably also want this to be `4.0-alpha-1` to remain compliant with maven versioning standards.